config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -315,9 +315,13 @@ public void onChangeEvent(ConfigurationChangeEvent event) {
                         event.setDataId(dataId).setNewValue(currentConfig).setOldValue(oldConfig);
                         listener.onChangeEvent(event);
                     }
-                    Thread.sleep(LISTENER_CONFIG_INTERNAL);
                 } catch (Exception exx) {
-                    LOGGER.error("fileListener execute error:{}", exx.getMessage(), exx);
+                    LOGGER.error("fileListener execute error:{}", exx.getMessage());
+                }
+                try {
+                    Thread.sleep(LISTENER_CONFIG_INTERNAL);
```

Comment:
Please help correct the spelling by the way.Thank you!
LISTENER_CONFIG_INTERNAL->LISTENER_CONFIG_INTERVAL

Reply:
> LISTENER_CONFIG_INTERVAL

ok

Reply:
cpu has behaved normally。

Reply:
> cpu has behaved normally。

If there are no questions, please reply to PTAL separately